MAINTAINERS

```diff
@@ -1,11 +1,4 @@
+The maintainers work across a series of projects at Weaveworks, for convenience they are in a github team
+@weaveworks/weavegitops-core-maintainers
```

Comment:
Does this team exist? I can only find @weaveworks/weaveworks-engineers-who-are-core-maintainers

Reply:
It does, but due to an error on my part the name isn't matching up, so I will fix in the internal system. This name is correct.

Reply:
fyi; I fixed this naming so it's correct now.